lib/floe/workflow/runner.rb

```diff
@@ -30,7 +30,38 @@ def for_resource(resource)
         end
       end
 
-      def run!(image, env = {}, secrets = {})
+      def run!(resource, env = {}, secrets = {})
```

Comment:
call this `run_async_shim!`?

so we can keep run_async and run together?

kubernetes could call into the shim

Reply:
the code lives on in kubernetes, so probably not necessary. we can address later